torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5263,29 +5233,19 @@ def c(t):
 
 
 def sample_inputs_linalg_qr(op_info, device, dtype, requires_grad=False, **kwargs):
-    """
-    This function generates input for torch.linalg.qr
-    The input is generated as the itertools.product of 'batches' and 'ns'.
-    """
+    # QR is just well defined when the matrix is full rank
+    make_fullrank = make_fullrank_matrices_with_distinct_singular_values
```

Comment:
haha, that's one way to shorten line length

Reply:
I should really put up a PR renaming this function...